oct/tests/deployment/deploy_app.py

```diff
@@ -2,14 +2,21 @@
 # pylint: disable=no-member
 # pylint: disable=invalid-name
 # pylint: disable=too-many-branches
+
 import os
+from enum import Enum
 from pyats.topology import Device
 from pyats.aetest import Testcase, test
 from pyats.utils.fileutils import FileUtils
 from requests.exceptions import ConnectTimeout
 from oct.tests import run_testcase
 
 
+class Status(Enum):
+    ON = 0
```

Comment:
@alexbeatnik I believe that if the app is working it has to be 
```suggestion
    ON = 2
```
and 0 for `OFF`.

Reply:
Fixed